### PR TITLE
Lodash error fix when choosing Grunt / Grunt Default Error

### PR DESCRIPTION
**Pull request by mcclaskiem**
Found when selecting grunt for workflow that the preprocessor evaluation failed due to an extra bracket in the if/else statement

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -303,7 +303,7 @@ KeystoneGenerator.prototype.tasks = function tasks () {
 
 		var gruntFiles = ['concurrent', 'express', 'node-inspector', 'nodemon'];
 
-		this.template('_Gruntfile.js', 'Gruntfile.js');
+		this.template('_gruntfile.js', 'gruntfile.js');
```
Comment (mxstbr) [inline app/index.js]: The `_Gruntfile` is uppercase, why would this need to be changed?

Comment (mcclaskiem) [inline app/index.js]: I changed the file case in my repo ignore that fix but the if else statement definitely needs to be merged

**Comment by mcclaskiem** [comment]
@mxstbr changed Gruntfile to uppercase

**Comment by pukeanddie** [comment]
thanks @mcclaskiem. Looks like this is the fix to the lodash error I was seeing!

**Comment by mcclaskiem** [comment]
- Also updated `registerTask` since `npm start` would fail with `default` task not found

- Reformatted preprocessor evaluation to make it more readable

**Comment by mxstbr** [comment]
Awesome, looks good to me!!

**Comment by A7bert** [comment]
Do I need to run the generator installation again? Or yo directly?

**Pull request merged**